packages/jest-worker/src/workers/messageParent.ts

```diff
@@ -7,31 +7,33 @@
 
 import {PARENT_MESSAGE_CUSTOM} from '../types';
 
-const isWorkerThread = () => {
+const isWorkerThread: boolean = (() => {
```

Comment:
only run this once instead of on each invocation of the `messageParent` function